cmd/cosign/cli/attest/attest_blob.go

```diff
@@ -127,7 +126,6 @@ func (c *AttestBlobCommand) Exec(ctx context.Context, artifactPath string) error
 		return errors.Wrap(err, "signing")
 	}
 
-	sig = []byte(base64.StdEncoding.EncodeToString(sig))
 	if c.OutputSignature != "" {
```

Comment:
IIRC we don't set a default value for `OutputSignature`. WDYT of defaulting to a file called `cosign_signature` or something so that it's always stored somewhere?

Reply:
It's always printing to STDOUT which is also the convention for `sign-blob` so people can just cat it into the file. I think it's OK like that?

Reply:
sgtm!